src/Analyzers/CSharp/Tests/UseCollectionInitializer/UseCollectionInitializerTests_CollectionExpression.cs

```diff
@@ -1242,8 +1242,10 @@ class C
             {
                 void M()
                 {
-                    List<int> c = [1,
-                        2];
+                    List<int> c =
+                    [
+                        1, 2
+                    ];
```

Comment:
lots of tests got better now that we rewrote the formatting logic for collection-exprs.